src/v/compat/begin_group_tx_compat.h

```diff
@@ -28,13 +28,15 @@ GEN_COMPAT_CHECK(
       json_write(pid);
       json_write(tx_seq);
       json_write(timeout);
+      json_write(tm_partition);
   },
   {
       json_read(ntp);
       json_read(group_id);
       json_read(pid);
       json_read(tx_seq);
       json_read(timeout);
+      json_read(tm_partition);
```

Comment:
this reminds me, is there anything on this change that should be exposed on the admin api, potentially via RPK?

Reply:
ah nvm. saw the change above.